plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineBundle.kt

```diff
@@ -63,27 +66,39 @@ open class GenerateGramineBundle @Inject constructor(
 
     override fun action() {
         enclaveJar.copyToOutputDir(GRAMINE_ENCLAVE_JAR)
+        // Building enclaves with Python has been temporarily disabled
+        // TODO: CON-1215 - Building enclaves with Python inside a Docker container
+        check(!pythonFile.isPresent) { "Building enclaves with Python has been disabled" }
```

Comment:
@shamsasari I still don't have clear what I need to do about Python now: my understanding is that I should not keep the Gramine-Non-docker flow, so should I just leave Python unusable with a TODO as it is here? Or even remove code that will be then readded (this sounds a bit weird).

Reply:
No, Python should still work, but not use the conclave-build container and instead rely on all the components installed on the local machine. You should still be able to run the PyTorch demo with your changes: https://github.com/R3Conclave/conclave-samples/tree/master/pytorch

Reply:
Does this mean that we will actually keep the current way of building enclaves then, just for Python?

Reply:
Correct. I assume this is not a problem, and just a matter of conditional statement.

Reply:
Ok, great, will readd back the conditional code